src/plugins/chart_expressions/expression_gauge/common/expression_functions/gauge_function.ts

```diff
@@ -94,7 +94,10 @@ export const gaugeFunction = (): GaugeExpressionFunctionDefinition => ({
       required: false,
     },
   },
-  fn(data, args) {
+  fn(data, args, handlers) {
+    args.ariaLabel =
```

Comment:
not sure that I like the idea of mutation `args` in that way. Am I right that with your changes `areaLabel` is in the acceptable arguments for that expression?

I think if we really need to take a description using that way you should: 

1) add `areaLabel` into expression definition
```
areaLabel: {
      types: ['string'],
      help: 'help text,
      required: false,
    },
```

2) update expression function 

```
 fn(data, args, handlers) {
    return {
      type: 'render',
      as: EXPRESSION_GAUGE_NAME,
      value: {
        data,
        args: {
          ...args,
          ariaLabel:
            args.ariaLabel ??
            handlers.variables?.embeddableTitle ??
            handlers.getExecutionContext?.()?.description,
        },
      },
    };
  },
```

@flash1293  do you have any thoughts?

Reply:
From my point of view we don't want this to be part of the expression. We can do this on the renderers and pass it to the component.

Reply:
I see only one possibility to get title it is pass it to handlers.variables or getExecutionContext, but I don't see option how we can fetch it on renderer because ExpressionRenderHandler doesn't include something like this.

Reply:
I agree with @alexwizp here - if we use the args to pass it in, we should make it a "real" argument. I don't think it hurts having it as part of the expression interface necessarily - this allows users in canvas to override it as well.
 @ppisljar @crob611 What do you think? Does it make sense to add the `ariaLabel` argument to all expression function?